src/components/content-tab-overview.tsx

```diff
@@ -161,17 +161,16 @@ function ShortcutsSection( { selectedSite }: Pick< ContentTabOverviewProps, 'sel
 				} catch ( error ) {
 					Sentry.captureException( error );
 					alert(
-						sprintf(
-							__( "Could not open the site code in %s. Please check if it's installed correctly." ),
-							'VS Code'
+						__(
```

Comment:
```suggestion
// translators: "VS Code" is the brand name for an IDE and does not need to be translated
						__(
```

What do you think about adding a translator note here to avoid these brand names being translated?